lib/pg_saurus/migration/set_role_method.rb

```diff
@@ -13,11 +13,53 @@ class << self
         #
         # @param role [String]
         def set_role(role)
+          if const_defined?("SeedMigrator".to_s) && self.ancestors.include?(SeedMigrator)
+            msg = <<~MSG
+              Use keep_default_role instead of set_role for data change migration #{self}\n
+              Example:\n
```

Comment:
Why is `\n` explicitly used here? It's sort of against the purpose of using a heredoc.